templates/instantanswer/new_ia.tx

```diff
@@ -154,3 +135,6 @@
         </ul>       
     </div>
 </div>
+<: if $c.session.ia_data { :>
+    <div class="hide" id="session_ia_data"></div>
+<: } :>
```

Comment:
I usually go for the inline syntax if I'm inside an element (eg if I have to decide which class to add), I think it's clearer.
In this case I have to decide whether to render an element or not, and since I'm outside of it I prefer this other style.